feature-libs/product/configurators/common/components/group-menu/configurator-group-menu.component.spec.ts

```diff
@@ -371,4 +376,16 @@ describe('ConfigurationGroupMenuComponent', () => {
     expect(configuratorGroupsService.isGroupVisited).toHaveBeenCalled();
     expect(configuratorGroupsService.getGroupStatus).toHaveBeenCalled();
   });
+
+  it('should verify whether the group is disable', () => {
```

Comment:
Small typo: should verify whether the group is disabled

Reply:
Done.